storage/src/lib.rs

```diff
@@ -81,3 +81,39 @@ pub use transaction::{abort, commit};
 
 pub type Data = Vec<u8>;
 pub type Result<T> = std::result::Result<T, Error>;
+
+#[cfg(test)]
+mod test {
+    use crate::{schema, traits::*};
+
+    struct MyColumn;
+    impl schema::Column for MyColumn {
```

Comment:
Based on the discussion yesterday, can we call "Column" a "DBIndex", then define it somewhere as clearly as `map<DBIndex, map<key, value>>`? To me, it's very important that people understand that none of this is even close to a relational database.

Reply:
Btw, this doesn't have to be changed in this PR. Maybe once this is merged, you can create a subseuent PR with only this change to separate them.

Reply:
I think I am leaning towards removing the schema machinery and partially avoid the issue of coming up with a good name for this.